source/connect/connection-options.txt

```diff
@@ -27,13 +27,21 @@ Connection Options
    Connection Pools </connect/connection-options/connection-pools>
    Connect from AWS Lambda <https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/>
 
-This section describes the MongoDB connection and authentication options
-available in the {+driver-short+}. You can configure your connection using either
-the connection URI or a ``MongoClientSettings`` object.
+--------
```

Comment:
S: I think it works but I dont think we need to add the title formatting above and below, just below
ie
```
Overview
--------
```

Reply:
yeah it makes the formatting confusing for the rest of the page. changed